### ISSUE TITLE
(error) FAILED to execute command: "php" "C:\Users\w11.vemto\AppData\Local\Programs\Vemto2\resources\static\VMTTL2"

### ISSUE THREAD

**Issue by vimitali368** [VemtoOrg/vemto2-issues#124]
**Error Message:**
(error) FAILED to execute command: "php" "C:\Users\w11.vemto\AppData\Local\Programs\Vemto2\resources\static\VMTTL2"

**Stack:**
```
Error: Command failed: "php" "C:\Users\w11.vemto\AppData\Local\Programs\Vemto2\resources\static\VMTTL2"

    at ChildProcess.exithandler (node:child_process:407:12)
    at ChildProcess.emit (node:events:527:28)
    at maybeClose (node:internal/child_process:1092:16)
    at ChildProcess._handle.onexit (node:internal/child_process:302:5)
```

**How to reproduce:**
1. 
2. 
3. 

**Expected behavior:**

**Actual behavior:**

**Additional context:**

---

Vemto 2.0.10
win32 10.0.25262
Locale: ru
App version: 2.0.10

**Comment by vimitali368** [comment]
All dependencies are green.

**Comment by TiagoSilvaPereira** [comment]
@vimitali368 is this a new project or an existing one? Does the problem happen if you try to create a Laravel project manually then connect it to Vemto? Does it happen if changing Schema Reader Mode to migration instead of database?

**Comment by vimitali368** [comment]
This is a new project.

**Comment by TiagoSilvaPereira** [comment]
@vimitali368, what happens if you go to your project folder on the command line and execute this?

```
php "C:\Users\w11.vemto\AppData\Local\Programs\Vemto2\resources\static\VMTTL2"
```

**Comment by TiagoSilvaPereira** [comment]
Comment by Rohan97 on it (from Discord):

> I found something related to this. When we generate the code first time this never happens. But after making changes and somehow if the anything is duplicate in the code base, for example if the class names are duplicate of same method names are there in a class that time this issue is happening. I observed this when I had same class name for two different listing pages. I resolved it and application works fine. You can take reference of these points to solve this I guess. Thank you.🙂

**Comment by TiagoSilvaPereira** [comment]
Fixed, please upgrade to 2.0.11
